webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -128,104 +134,96 @@ <h2>
                     </tr>
                 </thead>
                 <tbody>
-                {% for crash in tcbs %}
+                {% for topcrasher_stats in topcrashers_stats %}
```

Comment:
These two variable names differ in a single `s` in the middle of a long name. I'd much rather the variable be shorter and less similar. It'd make reading the rest of this code a lot easier for me. This is super easy to typo and make mistakes and the way we've got Jinja2 set up, we could miss typos.

Maybe something like:

```
{% for tcs in topcrashers_stats %}
```

if you're into abbreviations or perhaps:

```
{% for item in topcrashers_stats %}
```

if you're into that sort of thing.

Reply:
Or even:

```
{% for topcrashers_stats_item in topcrashers_stats %}
```

Also, this happens in a few other places in this PR.